frontend/tests/components/admin/report.test.tsx

```
@@ -26,52 +32,31 @@ jest.mock("sonner", () => ({
   },
 }));
 
+// Mock luxon DateTime
+jest.mock("luxon", () => ({
+  DateTime: {
+    fromISO: (date: string) => ({
```

ESLint

[error] 'date' is defined but never used.

(@typescript-eslint/no-unused-vars)